src/main/java/org/cyclops/integrateddynamics/core/network/PartNetworkElement.java

```diff
@@ -31,7 +32,7 @@
  */
 @Data
 public class PartNetworkElement<P extends IPartType<P, S>, S extends IPartState<P>> extends NetworkElementBase
-        implements IPartNetworkElement<P, S>, IEnergyConsumingNetworkElement {
+        implements IPartNetworkElement<P, S>, IEnergyConsumingNetworkElement, IEventListenableNetworkElement<P> {
```

Comment:
Not needed here, `IPartNetworkElement` already implements this.

Reply:
Removed.